dotcom-rendering/src/components/Branding.amp.tsx

```diff
@@ -57,8 +58,9 @@ export const Branding = ({ branding, pillar }: BrandingProps) => {
 			>
 				<amp-img
 					src={logo.src}
-					width={logo.dimensions.width}
-					height={logo.dimensions.height}
+					width="1.33"
+					height="1"
+					layout="responsive"
```

Comment:
That's the only way I've found [to make the image respect the width and the aspect ratio](https://amp.dev/documentation/components/amp-img#set-the-aspect-ratio). Please let me know if there's a better way!

Reply:
We know the dimensions server-side, so we need to keep those explicit values to ensure the aspect-ratio is correctly kept.

Reply:
Gotcha, thanks!